.github/workflows/ci.yaml

```diff
@@ -254,7 +254,7 @@ jobs:
           python3 -m pip install --upgrade pip
           python3 -m pip install --upgrade pytest
           python3 -m pip install -r requirements.txt
```

Comment:
This line should probably change to something like

`python3 -m pip install -e . --user`